tests/test/org/commcare/util/test/CaseAPITests.java

```diff
@@ -0,0 +1,117 @@
+package org.commcare.util.test;
+
+import org.commcare.api.persistence.SqlHelper;
+import org.commcare.cases.ledger.Ledger;
+import org.commcare.cases.model.Case;
+import org.javarosa.core.services.storage.util.DummyIndexedStorageUtility;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.DataInputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Vector;
+
+import static org.junit.Assert.fail;
+
+public class CaseAPITests {
+
+    Case a, b, c, d, e;
+
+    Ledger l;
+
+    DummyIndexedStorageUtility<Case> storage;
+    String owner;
+    String groupOwner;
+    String otherOwner;
+    Vector<String> groupOwned;
+    Vector<String> userOwned;
+
+
+    @Before
+    public void setUp() throws Exception {
+
+        storage = new DummyIndexedStorageUtility<Case>(Case.class, new PrototypeFactory());
```

Comment:
`storage` seems to be unused, does creating a new `DummyIndexStorageUtility` instance have side-effects that are needed here?